src/rules/custom-property-no-empty-line-before/__tests__/index.js

```diff
@@ -0,0 +1,73 @@
+import { mergeTestDescriptions, testRule } from "../../../testUtils"
+import { messages, ruleName } from ".."
+import rules from "../../../rules"
+
+const rule = rules[ruleName]
+
+const alwaysTests = {
+  accept: [ {
+    code: "a { --custom-color: pink; \r\n}",
```

Comment:
Lets go CSS code consistent with the examples in the docs, e.g. `--foo-bar: 1px`, throughout these tests. That'll make them a little easier to scan.